assets/src/blocks/Covers/ContentCovers.js

```diff
@@ -0,0 +1,87 @@
+import { useState, useLayoutEffect } from '@wordpress/element';
+
+const { __ } = wp.i18n;
+
+export const ContentCovers = ({ covers, initialRowsLimit, row, loadMoreCovers }) => {
+  const [amountPerRow, setAmountPerRow] = useState(4);
+
+  const updateRowAmount = () => setAmountPerRow(window.innerWidth >= 576 && window.innerWidth < 992 ? 3 : 4);
```

Comment:
A bit more readable if you move `window.innerWidth >= 576 && window.innerWidth < 992` into a function. Something like
```js
const updateRowAmount = () => setAmountPerRow(isMediumWindow ? 3 : 4);

Reply:
I guess you could then put `setAmountPerRow(isMediumWindow ? 3 : 4)` directly into `useLayoutEffect`

Reply:
I've moved the window width calculations (here and in TakeActionCovers) to separate function (i.e. `isMediumWindow`), but I still can't directly use it in `useLayoutEffect` as it needs to happen on window resize... I have however changed from `useLayoutEffect` to `useEffect` as it seems recommended to do so whenever possible!